macros/cross_db_utils/listagg.sql

```diff
@@ -0,0 +1,50 @@
+{% macro listagg(measure, delimiter_text, order_by_clause='', limit_clause='') -%}
```

Comment:
Looks like support for DISTINCT is missing here

Reply:
I believe DISTINCT is just part of the measure input. So if you wanted to use this macro with DISTINCT, you would write: `{{ dbt_utils.listagg('DISTINCT string_text', "','", "order by order_col") }}`.